Pod in Error STATUS

Hi,

After going trough the deployment steps my pod does not get out of error status. From the logs I see that is complains about the Cloudant function. I've run `npm install` and `npm run` on my local machine and didn't got the same error.

Gonne trough Cloudant steps and everything seems ok. Any ideas on how to fix it?

```
$ kubectl logs get-started-node-c689d597d-5btz9                                                                         
> get-started-node@0.1.1 start /app
> node server.js

/app/server.js:170
  cloudant = Cloudant(process.env.CLOUDANT_URL);
             ^

TypeError: Cloudant is not a function
    at Object.<anonymous> (/app/server.js:170:14)
    at Module._compile (module.js:577:32)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.runMain (module.js:611:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:160:9)
    at bootstrap_node.js:507:3

npm ERR! Linux 4.4.0-148-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.17.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! get-started-node@0.1.1 start: `node server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the get-started-node@0.1.1 start script 'node server.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the get-started-node package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node server.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs get-started-node
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls get-started-node
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /app/npm-debug.log
```

--- a/server.js
+++ b/server.js
@@ -167,6 +167,9 @@ if (appEnv.services['compose-for-mongodb'] || appEnv.getService(/.*[Mm][Oo][Nn][
      cloudant = Cloudant(appEnv.getService(/cloudant/).credentials);
   }
 } else if (process.env.CLOUDANT_URL){
+  // Load the Cloudant library.
+  var Cloudant = require('@cloudant/cloudant');
+
   cloudant = Cloudant(process.env.CLOUDANT_URL);
 }
 if(cloudant) {

